src/Goerli-DssSpell.sol

```diff
@@ -59,76 +36,28 @@ contract DssSpellAction is DssAction {
 
     function actions() public override {
 
-        // Add LerpFactory to ChainLog
-        DssExecLib.setChangelogAddress("LERP_FAB", LERP_FAB);
+        // Adjusting Auction Parameters for ETH-A, ETH-B, ETH-C, and WBTC-A
+        // https://vote.makerdao.com/polling/QmfGk3Dm?network=mainnet#poll-detail
 
-        // Offboard KNC Legacy Token
-        // https://vote.makerdao.com/polling/QmQ4Jotm?network=mainnet#poll-detail
-        DssExecLib.setIlkLiquidationPenalty("KNC-A", 0);
-        DssExecLib.linearInterpolation({
-            _name:      "KNC Offboarding",
-            _target:    DssExecLib.spotter(),
-            _ilk:       "KNC-A",
-            _what:      "mat",
-            _startTime: block.timestamp,
-            _start:       175 * RAY / 100,
-            _end:       5_000 * RAY / 100,
-            _duration:  60 days
-        });
+        // Decrease the Auction Price Multiplier (buf) for ETH-A, ETH-C, and WBTC-A vaults from 1.3 to 1.2
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-A",  12000);
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-C",  12000);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WBTC-A", 12000);
 
-        // Adopt the Debt Ceiling Instant Access Module (DC-IAM) for PSM-PAX-A
-        // https://vote.makerdao.com/polling/QmbGPgxo?network=mainnet#poll-detail
-        DssExecLib.setIlkAutoLineParameters({
-            _ilk:    "PSM-PAX-A",
-            _amount: 500 * MILLION,
-            _gap:     50 * MILLION,
-            _ttl:    24 hours
-        });
-        DssExecLib.setIlkAutoLineParameters({
-            _ilk:    "PSM-USDC-A",
-            _amount:  10 * BILLION,
-            _gap:    950 * MILLION,
-            _ttl:    24 hours
-        });
+        // Increase the Local Liquidation Limit (ilk.hole)
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-A",  40 * MILLION); //  from 30 Million DAI to 40 Million DAI
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-B",  25 * MILLION); //  from 15 Million DAI to 25 Million DAI
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-C",  30 * MILLION); //  from 20 Million DAI to 30 Million DAI
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-A", 25 * MILLION); //  from 15 Million DAI to 25 Million DAI
 
-        // G-UNI DAI/USDC
-        DssExecLib.setStairstepExponentialDecrease(MCD_CLIP_CALC_GUNI_A, 120 seconds, 9990);
+        // Increase the System Surplus Buffer from 60 Million DAI to 100 Million DAI
+        // https://vote.makerdao.com/polling/QmXcVy9R?network=mainnet#poll-detail
+        DssExecLib.setSurplusBuffer(100 * MILLION);
```

Comment:
Make sure to only do this is if going to the mainnet spell.

Reply:
Done in 060be11.